resources/qml/PrintSetupSelector/ProfileWarningReset.qml

```diff
@@ -91,16 +101,57 @@ Item
         iconSource: UM.Theme.getIcon("ArrowReset")
         anchors
         {
-            right: parent.right
+            right: buttonsSpacer.left
             verticalCenter: parent.verticalCenter
         }
 
-        color: UM.Theme.getColor("accent_1")
+        color: enabled ? UM.Theme.getColor("accent_1") : UM.Theme.getColor("disabled")
+        hoverColor: UM.Theme.getColor("primary_hover")
+
+        enabled: Cura.MachineManager.hasCustomQuality || Cura.SimpleModeSettingsManager.isProfileCustomized
+        onClicked: Cura.MachineManager.resetToUseDefaultQuality()
 
-        onClicked:
+        UM.ToolTip
         {
-            Cura.MachineManager.resetToUseDefaultQuality()
+            visible: parent.hovered
+            y: parent.y + parent.height + UM.Theme.getSize("default_margin").height
+            targetPoint: Qt.point(parent.x, Math.round(parent.y + parent.height / 2))
+            tooltipText: catalog.i18nc("@info", "Reset to defaults.")
         }
     }
 
-}
\ No newline at end of file
+    // Spacer
+    Item
+    {
+        id: buttonsSpacer
+        width:  UM.Theme.getSize("action_button_icon").height
+        anchors.right: compareAndSaveButton.left
+    }
+
+    UM.SimpleButton
+    {
+        id: compareAndSaveButton
+        height: UM.Theme.getSize("action_button_icon").height
+        width: height
+        iconSource: UM.Theme.getIcon("Save")
+        anchors
+        {
+            right: parent.right
+            verticalCenter: parent.verticalCenter
+        }
```

Comment:
```suggestion
        anchors
        {
            right: parent.right
            verticalCenter: parent.verticalCenter
        }
        iconSource: UM.Theme.getIcon("Save")
```

Reply:
I don't ofter use

anchors

{

 etc

}

Since it only really make the code more consice/readable when you are hitting 6 assignments to anchors.

Reply:
I tend to for larger sets of anchors, as it makes it easier to ignore all of them (eg; it's clear that we're talking about a group of anchors in once glance)